src/cdk/drag-drop/drag.ts

```diff
@@ -266,30 +269,36 @@ export class CdkDrag<T = any> implements AfterViewInit, OnDestroy {
     // element to be in the proper place in the DOM. This is mostly relevant
     // for draggable elements inside portals since they get stamped out in
     // their original DOM position and then they get transferred to the portal.
-    this._ngZone.onStable.asObservable().pipe(take(1)).subscribe(() => {
-      const rootElement = this._rootElement = this._getRootElement();
-      rootElement.addEventListener('mousedown', this._pointerDown, passiveEventListenerOptions);
-      rootElement.addEventListener('touchstart', this._pointerDown, passiveEventListenerOptions);
-      toggleNativeDragInteractions(rootElement , false);
-    });
+    this._rootElementInitSubscription = this._ngZone.onStable.asObservable()
```

Comment:
Could to use `takeUntil` with ` _destroyed` subject here?

Reply:
I could, but then we'd have the `_destroyed` only for this one case.